src/Piral.Blazor.Core/Program.cs

```diff
@@ -31,6 +32,7 @@ public static async Task Main(string[] args)
         private static void Configure(WebAssemblyHost host)
         {
             JSBridge.Configure(host.Services.GetRequiredService<HttpClient>());
+            Extensions.Configure(host.Services.GetService<ILoggerFactory>());

```

Comment:
This is an anti-pattern. The only reason to do it with the bridge is that the bridge is already an anti pattern and this was the only way.

For the above you have better options. If you *really* want a logger in there (which you shouldn't) then bring it in as an argument. Then its up to the caller. In general, however, it should be the caller of an extension to do the logging. Extensions are low-level functions that don't do the logging (or error handling). They define the working parts.